### PR TITLE
Execute fix

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: src/clients/reipl_bootloader_finish.rb [hunk 1]
```diff
@@ -50,7 +50,7 @@ def main
             res
           end
           # zkvm require change of IPL (bnc#943582)
-          zkvm = Yast::WFM.Execute(".local.bash", "egrep 'Control Program: KVM' /proc/sysinfo") == 0
+          zkvm = Yast::WFM.Execute(Yast::Path.new(".local.bash"), "egrep 'Control Program: KVM' /proc/sysinfo") == 0
```
Comment (aschnell) [inline src/clients/reipl_bootloader_finish.rb]: Isn't 'path(".local.bash")' the common used expresssion?

Comment (jreidinger) [inline src/clients/reipl_bootloader_finish.rb]: yes, but I am not sure here if it lives in Yast namespace, so I do not want to risk, as I do not have zKVM for testing

File: package/yast2-reipl.changes [hunk 2]
```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Fri Nov 13 15:01:40 UTC 2015 - jreidinger@suse.com
+
+- fix error message in zKVM (bnc#954952)
```
Comment (mvidner) [inline package/yast2-reipl.changes]: > Fixed an Internal Error on z/KVM (bnc#954952).

**Comment by mvidner** [comment]
LGTM, optionaly improve the .changes message.

File: package/yast2-reipl.changes (hunk 2 above)
Comment (jreidinger) [inline package/yast2-reipl.changes]: ok, sounds better.

**Pull request merged**